cron/exhibits_cron

```diff
@@ -0,0 +1,2 @@
+* * * * * root find /tmp -type f -atime +7 -delete
```

Comment:
Fwiw, it seems strange to have 2 different ways of setting up cron jobs, so I'll probably move this to schedule.rb so it's handled by the whenever gem. I'll probably ask you in our upcoming meeting what would need to change if we get rid of this cron file!